book/src/install.md

```diff
@@ -134,11 +134,15 @@ pacman -S mingw-w64-ucrt-x86_64-helix
 
 ## Building from source
 
-Clone the repository:
+According to the Unix Filesystem Hierarchy Standard the Helix source can be
+installed in **/usr/local/src** or **/opt**, however we would prefer to install
+Helix as a regular user without **sudo** access, so we can create a
+**$HOME/src** directory.
```

Comment:
Since we're not going to use FHS guideless I don't think we need to mention it

Reply:
Suggestions adopted and committed @the-mikedavis